Add two new sections + short descriptions to the settings overview (#6012)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/README.md

* [Public Access](./#public-access)
* [Deployment Webhooks](./#webhooks)
* [Secure Sockets Layer (SSL) certificates](./#certificates-only-available-on-professional-or-enterprise-plan)

> * [CI/CD Flow.](./#umbracocicd)

Reviewer: @jesp209i didn't notice this at first, but how come you are removing CI/CD from the overview? 🤔

You can see metrics related to the overall health and performance of the Azure app service hosting the live environment of your solution.

> ## Automatic Upgrades

Reviewer: I also think it makes sense to link the the page that explains how to enable this feature 😄

In that case it is this page:  https://docs.umbraco.com/umbraco-cloud/product-upgrades/minor-upgrades

Author: link added

We handle minor and patch upgrades for the Umbraco components used by Umbraco Cloud, so you don't have to. 

> Here you can control if you want to opt-in or out of automatic minor upgrades. 

Reviewer: ```suggestion
From the Automatic Upgrades page, you can control if you want to opt in or out of automatic minor upgrades. 
```

New projects are opt-in by default.

## Backups

> You can handle backups from this setting. You can create a database backup of one or more of your cloud environments. 

Reviewer: ```suggestion
With this setting, it is possible to create a database backup of one or more of your cloud environments. 
```

> (Coming soon) It is also possible to upload your own '.bacpac' file.

Reviewer: Maybe let's wait to add this to the page until the feature has been released on Umbraco Cloud 😊

The same is also the case with the sentence below.

> 
> (Coming soon) It is also possible to upload your own '.bacpac' file.
> 
> (Coming soon) You can also restore a bacpac file to an environment. 
> 

Author: ```suggestion

```
